front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -452,6 +453,8 @@ const AdminProjectsProjectGeneral = () => {
     !isNilOrError(project) ? project : null
   );
 
+  const projectCardShouldBeSaved = projectCard ? !projectCard.remote : false;
```

Comment:
```suggestion
  const projectCardImageShouldBeSaved = projectCard ? !projectCard.remote : false;
```

Reply:
But I use just the word `card` (w/o `image`) in all other variables. Shouldn't it be consistent?